test/framework/clean_up.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/gomega"
+	corev1 "sigs.k8s.io/cluster-api/api/v1alpha3"
```

Comment:
`clusterv1` instead? corev1 will probably get mixed up with `k8s.io/api/core/v1`.

Reply:
I don't care, I've seen it go both ways, but clusterv1 is less ambiguous than corev1. I'll change these